global.json

```
@@ -1,6 +1,11 @@
 {
+  "sdk": {
+    "version": "6.0.300",
+    "rollForward": "latestMinor",
+    "allowPrerelease": true
+  },
   "tools": {
-    "dotnet": "6.0.100-rc.2.21478.25"
+    "dotnet": "6.0.300-preview.22173.2"
```

Error is:
```
      A compatible installed .NET SDK for global.json version [6.0.300] from [/Users/builder/azdo/_work/1/s/global.json] was not found.
      Install the [6.0.300] .NET SDK or update [/Users/builder/azdo/_work/1/s/global.json] with an installed .NET SDK:
        6.0.201 [/Users/builder/azdo/_work/_tool/dotnet/sdk]
```

Maybe try latest stable?
```suggestion
    "dotnet": "6.0.201"
```